ledger/storage/record/registry.go

```diff
@@ -0,0 +1,21 @@
+package record
+
+var registry = map[TypeID]Record{}
+
+// Register makes provided record serializable. Should be called for each record in init().
+func register(id TypeID, r Record) {
+	if _, ok := registry[id]; ok {
+		panic("duplicate record type")
+	}
+
+	registry[id] = r
+}
+
+// Registry returns records by type.
+func Registry() map[TypeID]Record {
```

Comment:
I suppose, it's better for us to rename this method. Smth like GetRegistrated or Registrated